client/components/social-buttons/google.js

```diff
@@ -154,34 +154,40 @@ class GoogleLoginButton extends Component {
 		const isDisabled = Boolean( this.state.isDisabled || this.props.isFormDisabled || this.state.error );
 
 		const { children } = this.props;
+		let customButton = null;
+
 		if ( children ) {
 			const childProps = {
+				onClick: this.handleClick,
 				onMouseOver: this.showError,
 				onMouseOut: this.hideError,
-				onClick: this.handleClick,
+				className: classNames( { disabled: isDisabled } ),
```

Comment:
Minor: If we're alphabetizing the key names, might as well shoot className to the top 😛